inspirehep/modules/workflows/static/js/inspire_workflows_ui/templates/details_decision_authors.html

```diff
@@ -1,4 +1,7 @@
 <div ng-if="vm.record._extra_data._action === 'author_approval' && vm.record._workflow.status === 'HALTED'">
+<button class="btn btn-info"
+        ng-click="Utils.redirectToReviewSubmission(vm.record.id)">Review Submission
```

Comment:
In principle I was thinking of leaving it in the same place you have it now, just a more generic `Utils.redirect` instead of `Utils.redirectToReviewSubmission`